plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -138,7 +137,7 @@ protected Object readResolve() throws IOException {
 
     private static final String KEEP_STEP_ARGUMENTS_PROPERTYNAME = (DSL.class.getName()+".keepStepArguments");
 
-    private static boolean isKeepStepArguments = StringUtils.isEmpty(System.getProperty(KEEP_STEP_ARGUMENTS_PROPERTYNAME))
+    private static boolean isKeepStepArguments = System.getProperty(KEEP_STEP_ARGUMENTS_PROPERTYNAME) == null || System.getProperty(KEEP_STEP_ARGUMENTS_PROPERTYNAME).isEmpty()
             || Boolean.parseBoolean(System.getProperty(KEEP_STEP_ARGUMENTS_PROPERTYNAME));
```

Comment:
Actually this should be replaced with `SystemProperties.getBoolean` (or `Boolean.getBoolean`).